[2024-05-03T02:09:21.049+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-03T02:09:21.089+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-04-21T12:00:00+00:00 [queued]>
[2024-05-03T02:09:21.106+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-04-21T12:00:00+00:00 [queued]>
[2024-05-03T02:09:21.112+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-03T02:09:21.162+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_weather_data> on 2024-04-21 12:00:00+00:00
[2024-05-03T02:09:21.174+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=427) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-03T02:09:21.177+0000] {standard_task_runner.py:63} INFO - Started process 446 to run task
[2024-05-03T02:09:21.180+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_parse_weather_data', 'fetch_and_save_weather_data', 'scheduled__2024-04-21T12:00:00+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/json_dag.py', '--cfg-path', '/tmp/tmpg7n_qbqw']
[2024-05-03T02:09:21.191+0000] {standard_task_runner.py:91} INFO - Job 249: Subtask fetch_and_save_weather_data
[2024-05-03T02:09:21.287+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-04-21T12:00:00+00:00 [running]> on host e8691f41e037
[2024-05-03T02:09:21.453+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_parse_weather_data' AIRFLOW_CTX_TASK_ID='fetch_and_save_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-21T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-21T12:00:00+00:00'
[2024-05-03T02:09:21.458+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-03T02:09:22.797+0000] {logging_mixin.py:188} INFO - Data has been successfully saved to the JSON file
[2024-05-03T02:09:22.798+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-03T02:09:22.799+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-03T02:09:22.810+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_and_parse_weather_data, task_id=fetch_and_save_weather_data, execution_date=20240421T120000, start_date=20240503T020921, end_date=20240503T020922
[2024-05-03T02:09:22.880+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-03T02:09:22.901+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-03T02:09:22.904+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
